src/build/dotnet/jobs/nuget-package.job.yaml

```diff
@@ -69,6 +69,10 @@ jobs:
             --configuration Release
             --no-restore
 
+      - template: /src/build/common/steps/license-checks.steps.yaml
```

Comment:
Does this need to happen after the build? It can't go up at the top with the dependency check? Feels a bit random here between Build and Test